analysis/museumdraw/analysis1_nov25_vssAbstract.Rmd

```diff
@@ -187,57 +244,48 @@ joint[,colInd]=scale(joint[,colInd])
 
+``` {r}
+mod_covariates_age <- glmer(correct ~ (scale(age) + 
+                          scale(draw_duration) + 
+                          scale(num_strokes))^2 +
+                        (1|session_id) + 
+                        (1 | category), 
+      data = joinedRatings,  
+      family = "binomial")
+
+kable(summary(mod_covariates_age)$coef, digits = 3)
+```
 ## VSS ABSTRACT DRAFT
-Title options:
-"Drawings convey object category information by middle childhood"
 
-"Drawings as a window into the development of object category representations"
+**NOTE: COEFFICIENTS NOT UPDATED**
 
-"Development of ability to depict object categories"
+Title options:
+"Drawings as a window into the development of object category representations"
 
 Abstract:
 Drawing is a powerful tool for communicating concepts in visual form — a few well-placed strokes can convey the identity of a person, object, or scene. Prior work has found that deep neural network models of the ventral stream trained purely on photographs can also recognize drawings by nonexpert adults, reflecting concordance in abstract representations of object categories in drawings and photos at higher layers in these models (Fan, Yamins, & Turk-Browne, 2015). How do ordinary people become so effective at producing recognizable drawings? Here we examine the trajectory of this learning during childhood. Children (N = `r numDrawers`, *M* = `r avgAge` years, range 4-10 years) participated in an iPad-based drawing game where they were prompted with a verbal cue to draw one of sixteen familiar objects (e.g., “Can you draw a cup?”). Children drew each object category for 30 seconds, after which they were prompted to either make another drawing or to stop drawing altogether. Afterwards, a group of naive adults (N = `r length(unique(r$workerid))`) guessed the identity of each drawn object(`r numDrawings` drawings). Using a linear mixed-effect modeling approach, we found that recognizability of drawings increased reliably with age (b = `r ageBeta`, SE = `r ageSE`, Z = `r ageZ`), accounting for variation across categories and children and the amount of time spent drawing, number of strokes, and total ink used  (% drawings recognized; chance = 4.8%; $M_{4yrs}$ = `r ageCorrOut[1]`%, $M_{5yrs}$ = `r ageCorrOut[2]`%, $M_{6yrs}$ = `r ageCorrOut[3]`%, $M_{7yrs}$ = `r ageCorrOut[4]`%, $M_{8yrs}$ = `r ageCorrOut[5]`%, $M_{9yrs}$ = `r ageCorrOut[6]`%, $M_{10yrs}$ = `r ageCorrOut[7]`%); removing age from the model significantly decreased model fit ($X^2$(1) = `r round(modelOut$Chisq[2],2)`, *p* = `r pOut` ). These results suggest that the capacity to quickly produce graphical representations that communicate object category information is highly developed by middle childhood. More broadly, these findings point to visual production tasks as a promising avenue for examining the development of object category representations.
```

Comment:
I think you might alter the story a little bit to be "independent of these low-level covariates, kids are developing skill" maybe? that's slightly more interesting than just the first order "they're getting better."